src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -12,8 +12,11 @@
 use Roave\BetterReflection\Reflection\ReflectionClassConstant as BetterReflectionClassConstant;
 use Roave\BetterReflection\Reflection\ReflectionMethod as BetterReflectionMethod;
 use Roave\BetterReflection\Reflection\ReflectionObject as BetterReflectionObject;
+use Roave\BetterReflection\Reflection\ReflectionProperty as BetterReflectionProperty;
+use Webmozart\Assert\Assert;
```

Comment:
Oooof, we don't have the `webmozart/assert` dependency: will get rid of this one.

Reply:
Actually, we don't have either assertion library included as a `require` in `composer.json`, wonder if it's worth adding?

Reply:
probably not for this specific change

Reply:
OK :+1: